prog/vm/github_runner.rb

```diff
@@ -393,6 +393,19 @@ def setup_info
           Clog.emit("Failed to move serial.log or running journalctl") { github_runner }
         end
       end
+
+      # For analyzing the Docker Hub rate limit, we check the quota and log the
+      # remaining limit.
+      begin
+        docker_quota_limit_command = <<~COMMAND
+          TOKEN=$(curl "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull" | jq -r .token)
+          curl --head -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest | grep ratelimit
```

Comment:
```suggestion
          TOKEN=$(curl -s "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull" | jq -r .token)
          curl -s --head -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest | grep ratelimit
```

It removed unnecessary progress logs.

![CleanShot 2025-03-27 at 06 00 47@2x](https://github.com/user-attachments/assets/4ee2b189-6983-4afb-88fe-f369387e3680)